reference/opt/shaders-msl/asm/frag/locations-components.asm.frag

```diff
@@ -11,10 +11,10 @@ struct main0_out
 struct main0_in
 {
     float2 m_2 [[user(locn1)]];
-    float m_3 [[user(locn1)]];
+    float m_3 [[user(locn1_2)]];
```

Comment:
Does "locn" mean anything in MSL? Or does MSL link user attributes by name?

Reply:
Like HLSL with its semantics, MSL links user attributes by name.